packages/server/src/middleware/authenticated.js

```diff
@@ -17,47 +19,35 @@ module.exports = async (ctx, next) => {
   const appToken = ctx.cookies.get("budibase:token")
   const builderToken = ctx.cookies.get("builder:token")
 
-  if (builderToken) {
-    try {
-      const jwtPayload = jwt.verify(builderToken, ctx.config.jwtSecret)
-      ctx.auth = {
-        apiKey: jwtPayload.apiKey,
-        authenticated: jwtPayload.accessLevelId === BUILDER_LEVEL_ID,
-      }
-      ctx.user = {
-        ...jwtPayload,
-        accessLevel: await getAccessLevel(
-          jwtPayload.instanceId,
-          jwtPayload.accessLevelId
-        ),
-      }
-    } catch (_) {
-      // empty: do nothing
-    }
-
-    await next()
-    return
+  let token
+  // if running locally in the builder itself
+  if (!environment.CLOUD && !appToken) {
```

Comment:
I am going to pull this down and lay with with it. But I think that this might break for us when testing with `yarn dev` in the browser (and `yarn build:electron` is currently broken for windows)... e.g.
1. Click preview > browser gets appToken
2. Back into the builder (still in browser) > all calls will be authenticating as appToken

I will check it out, but I think this is what will happen

If we are happy with this, then NAB

Reply:
Good point - we could possibly just add a condition for electron if its a big issue

Reply:
Check this, this is the case. However, it currently has no effect, as ADMIN users have full access to all endpoints (see comment below)